tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -488,6 +488,7 @@ public void uploadAndSaveRunInfo() {
         if (getTotalWallClockTimeInISO861Standard() != null) {
             runMetrics.setExecutionTime(getTotalWallClockTimeInISO861Standard());
         }
+        runMetrics.setDateExecuted(workflowStartTime.toString());
```

Comment:
I think this might fail when submitted. `workflowStartTime` is a `Date` object, but `dateExecuted` should be in ISO 8601 date format:
https://github.com/dockstore/dockstore/blob/20bc42357a5d1a1d1ff798a8bf969e1f8043dd4a/dockstore-webservice/src/main/java/io/dockstore/webservice/core/metrics/Execution.java#L38

I think `workflowStartTime.toInstant().toString()` may work because [Instant.toString()](https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#toString--) uses ISO-8601 representation